model/version.go

```diff
@@ -599,9 +599,6 @@ func GetVersionsWithOptions(projectName string, opts GetVersionsOptions) ([]Vers
 	if err := db.Aggregate(VersionCollection, pipeline, &res); err != nil {
 		return nil, errors.Wrapf(err, "error aggregating versions and builds")
 	}
-	if len(res) == 0 {
```

Comment:
why was this removed?

Reply:
waiting on this question

Reply:
i think because we would be returning the same thing if we return in this if statement or out of it there's no point in doing the check

Reply:
oh, true! thanks!